src/subdomains/generic/kyc/services/kyc.service.ts

```diff
@@ -122,6 +128,39 @@ export class KycService {
     }
   }
 
+  // TODO: Remove temporary cron job
+  @Cron(CronExpression.EVERY_MINUTE)
+  async storeExistingKycFiles(): Promise<void> {
+    try {
+      this.logger.info('Starting Cron Job to store existing KYC files');
+
+      const allUserData = await this.userDataService.getAllUserData();
+
+      for (const userData of allUserData) {
+        const existingFiles = await this.documentService.listUserFiles(userData.id);
+
+        for (const existingFile of existingFiles) {
+          try {
+            const exampleKycFile = {
+              name: existingFile.name,
+              type: existingFile.type,
+              protected: false,
+              userData: userData,
+            };
+
+            await this.kycFileService.createKycFile(exampleKycFile);
+          } catch {
+            this.logger.error(`Failed to store existing KYC file ${existingFile.name} for user ${userData.id}`);
+          }
+        }
+      }
+
+      this.logger.info('Successfully stored existing KYC files in the database');
+    } catch (error) {
+      this.logger.error('Failed to store existing KYC files:', error);
+    }
+  }
```

Comment:
We need a way to find out which users have already been "synced", to avoid data duplication. Maybe filter out all users, that already have an entry in KYC file table?